sections/02_intro.md

```diff
@@ -1,31 +1,27 @@
 ## Introduction
 
-Biology and medicine are rapidly becoming particularly data-intensive field with
+Biology and medicine are rapidly becoming particularly data-intensive with
 respect to both research and practice. A recent comparison of genomics with
 social media, online videos and other data-intensive scientific disciplines
 suggested that the field of genomics alone would equal or surpass other fields
 in data generation and analysis within the next decade
 [@doi:10.1371/journal.pbio.1002195]. These data present new opportunities, but
-also new challenges. We expect that algorithms to automatically extract
-meaningful patterns and provide sufficient context to enable us to act will be
-required.
+also new challenges. We foresee a great need for algorithms that can automatically extract meaningful patterns while providing sufficient context to act, all within data constrained and privacy critical environments.
```

Comment:
(this incorporates suggestions made by @gwaygenomics on my draft and would supercede the changes we merged yesterday)